src/dataset/processing.py

```diff
@@ -228,6 +228,53 @@ def save_chunks(chunks, info, save_dir, use_tqdm: bool) -> None:
             y=normalize(chunk_y.reshape(-1, 1)).squeeze(),
         )
 
+def split_chunks(all_chunks: list, file_info: list, chunk_size: int, test_size: float, val_size: float) -> tuple:

```

Comment:
To typowanie trochę takie o dupę potłuc jak nie ma info o tym jakie typy są w obiekcie iterowalnym czyli np. List[int] czy Tuple[np.ndarray, int].
Ale to już takie tylko trochę złośliwe czepialstwo XD